service/src/main/java/bio/terra/policy/service/policy/PolicyUnknown.java

```diff
@@ -70,4 +70,15 @@ public PolicyInput combine(PolicyInput dependent, PolicyInput source) {
   public PolicyInput remove(PolicyInput target, PolicyInput removePolicy) {
     return null;
   }
+
+  /**
+   * Unknown policies will be seen as valid.
```

Comment:
I think that we should disallow unknown policies for the time being. It will always be an error right now.
When we have a concrete use case for allowing unknown policies, we can change this.
wdyt?

Reply:
I was debating that. The reason I didn't was that we have a lot of tests that do merging/linking/combining that use unknown policies. So I'd break those tests. I could rework them to use regions, but it's not a simple conversion since regions combine differently.
It probably is the right time to do that though. I'll see how much work it really is.